src/components/Article.js

```diff
@@ -87,12 +88,16 @@ const ArticleSection = ({
 const ArticleInner = ({ lang, articleTitle, initialSubTitle }) => {
   const i18n = useI18n()
   const containerRef = useRef()
-  const article = useArticle(lang, articleTitle)
+  const [article, loadArticle] = useArticle(lang, articleTitle)
 
-  if (!article) {
+  if (article.loading) {
```

Comment:
we can keep this form without the `loading` key, so the loading animation shows by default when fetching the event.

Reply:
Done